src/Tribe/Linked_Posts/Chooser_Meta_Box.php

```diff
@@ -0,0 +1,226 @@
+<?php
+
+/**
+ * Class Tribe__Events__Linked_Posts__Chooser_Meta_Box
+ *
+ * Handles the Organizer section inside the Events meta box
+ */
+class Tribe__Events__Linked_Posts__Chooser_Meta_Box {
+	/**
+	 * @var WP_Post
+	 */
+	protected $event;
+
+	/**
+	 * @var Tribe__Events__Main
+	 */
+	protected $tribe;
+
+	/**
+	 * @var Tribe__Events__Linked_Posts
+	 */
+	protected $linked_posts;
+
+	/**
+	 * @var string
+	 */
+	protected $post_type;
+
+	/**
+	 * @var string
+	 */
+	protected $singular_name;
+
+	public function __construct( $event = null, $post_type = null ) {
+		$this->tribe        = Tribe__Events__Main::instance();
+		$this->linked_posts = Tribe__Events__Linked_Posts::instance();
+		$this->post_type = $post_type;
+		$this->singular_name = $this->linked_posts->linked_post_types[ $this->post_type ]['labels']['singular_name'];
+		$this->get_event( $event );
+	}
+
+	/**
+	 * Work with the specifed event object or else use a placeholder if we are in
+	 * the middle of creating a new event.
+	 *
+	 * @param null $event
+	 */
+	protected function get_event( $event = null ) {
+		if ( is_null( $event ) ) {
+			$event = $GLOBALS['post'];
+		}
+
+		if ( is_numeric( $event ) ) {
+			$event = WP_Post::get_instance( $event );
+		}
+
+		if ( $event instanceof stdClass || is_array( $event ) ) {
+			$event = new WP_Post( (object) $event );
+		}
+
+		if ( ! $event instanceof WP_Post ) {
+			$event = new WP_Post( (object) array( 'ID' => 0 ) );
+		}
+
+		$this->event = $event;
+	}
+
+	/**
+	 * Render the organizer chooser section for the events meta box
+	 *
+	 */
+	public function render() {
+		$this->render_dropdowns();
+		$this->render_add_post_button();
+
+		/**
+		 * Make this Template filterable, used for Community Facing templates
+		 *
+		 * @var string $file_path
+		 */
+		include apply_filters( 'tribe_events_multiple_linked_post_template', $this->tribe->pluginPath . 'src/admin-views/linked-post-meta-box.php' );
+	}
+
+	/**
+	 * displays the saved organizer dropdown in the event metabox
+	 * Used to be a PRO only feature, but as of 3.0, it is part of Core.
+	 *
+	 */
+	public function render_dropdowns() {
+		$post_id = $this->event->ID;
+
+		$current_linked_posts = get_post_meta( $post_id, $this->linked_posts->get_meta_key( $this->post_type ), false );
+
+		if ( $this->use_default_post( $current_linked_posts ) ) {
+			$current_linked_posts = apply_filters( 'tribe_events_linked_post_default', array(), $this->post_type );
+		}
+
+		$current_linked_posts = (array) apply_filters( 'tribe_display_event_linked_post_dropdown_id', $current_linked_posts, $this->post_type );
+
+		/* if the user can't create organizers, then remove any empty values
+		   from the $current_organizers array. This prevents the automatic
+		   selection of an organizer every time the event is edited. */
+		$linked_post_pto = get_post_type_object( $this->post_type );
```

Comment:
This method was a lift of what was already in place for the Organizer dropdown, so I think we're safe on this front.